### ISSUE TITLE
dependency not included in appimage

### ISSUE THREAD

**Issue by sudoshindo** [jsnli/Samira#5]
It's telling me that **glibc 2.39** is not found.

This problem doesn't happen on the original SamRewritten, please fix.

**Comment by jsnli** [comment]
Are you in the right directory? That error indicates the AppImage isn't present and is unrelated to dependencies.

**Comment by sudoshindo** [comment]
Yes, I know how to run AppImages and other executables in the terminal.

**Comment by sudoshindo** [comment]
Sorry I took the wrong screenshot because I opened another terminal window to run the commands.

It's telling me that **glibc 2.39** is missing

**Comment by sudoshindo** [comment]
Realized the screenshot mistake.
Removed the image and just wrote out what error I got.
My bad.

**Comment by jsnli** [comment]
Could you share what distro and glibc version you're on? `dpkg -l libc6`

**Comment by sudoshindo** [comment]
I'll send it when I get back home. I need to go somewhere at the moment, I already turned off the PC.

**Comment by sudoshindo** [comment]
Fedora 40
```
ldd --version
ldd (GNU libc) 2.38
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

**Comment by jsnli** [comment]
Compiling Tauri on Ubuntu 22.04.5 would widen support but is currently blocked by libwebit2gtk-4.1-0 [issues](https://github.com/gitbutlerapp/gitbutler/issues/5282). This will probably be fixed in the future by it's maintainers. For now you'll need to upgrade your system.

**Comment by KenKaneki18** [comment]
> Compiling Tauri on Ubuntu 22.04.5 would widen support but is currently blocked by libwebit2gtk-4.1-0 [issues](https://github.com/gitbutlerapp/gitbutler/issues/5282). This will probably be fixed in the future by it's maintainers. For now you'll need to upgrade your system.

Hey, I'm truly sorry for just butting in. I don't know how to contact users on here. I just have a question. Is your program also compatible with non Steam games or does it only work for games you purchased on Steam?
Thanks in advance

**Comment by jsnli** [comment]
No, Samira only interfaces with games purchased on Steam.
